packages/samples/sources/programmability/module-initializer.move

```diff
@@ -7,30 +7,30 @@ module book::store {
     /// the store.
     public struct StoreOwnerCap has key, store { id: UID }
 
-    /// The store itself, one and only, created in the `init` function.
```

Comment:
A small note, calling this module `store` might be a bit confusing for folks learning the language, since it is unrelated to the `store` ability.

Reply:
Thanks, patched! Now store -> shop